src/Tasks/Microsoft.NET.Build.Extensions.Tasks/Microsoft.NET.Build.Extensions.Tasks.csproj

```diff
@@ -0,0 +1,75 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- This is a smaller build of Microsoft.NET.Build.Tasks to be used outside
+       of SDK projects -->
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <RootNamespace>Microsoft.NET.Build.Tasks</RootNamespace>
+    <TargetFrameworks>netcoreapp1.0;net46</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp1.0</TargetFrameworks>
+    <GenerateDependencyFile>false</GenerateDependencyFile>
+    <OutputPath>$(OutputPath)Sdks\Microsoft.NET.Sdk\msbuildExtensions\Microsoft.NET.Build.Extensions\tools</OutputPath>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <DefineConstants>$(DefineConstants);EXTENSIONS</DefineConstants>
+  </PropertyGroup>
+
+  <ItemDefinitionGroup>
+    <PackageReference>
+      <PrivateAssets>All</PrivateAssets>
+      <Publish>true</Publish>
+    </PackageReference>
+  </ItemDefinitionGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Build.Framework" Version="$(MsBuildPackagesVersion)" ExcludeAssets="Runtime" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MsBuildPackagesVersion)" ExcludeAssets="Runtime" />
+    <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataVersion)" Condition=" '$(TargetFramework)' == 'netcoreapp1.0' " />
+    <PackageReference Include="System.Diagnostics.FileVersionInfo" Version="4.0.0" Condition=" '$(TargetFramework)' == 'netcoreapp1.0' " />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="**\*.cs" />
+    <SharedCompile Include="BuildErrorException.cs" />
+    <SharedCompile Include="DiagnosticsHelper.cs" />
+    <SharedCompile Include="DiagnosticMessageSeverity.cs" />
+    <SharedCompile Include="ItemUtilities.cs" />
+    <SharedCompile Include="MetadataKeys.cs" />
+    <SharedCompile Include="NETSdkError.cs" />
+    <SharedCompile Include="NuGetUtils.cs" />
+    <SharedCompile Include="TaskBase.cs" />
+    <Compile Include="@(SharedCompile->'..\Microsoft.NET.Build.Tasks\%(Identity)')">
+      <Link>%(FileName)%(Extension)</Link>
+    </Compile>
+    <Compile Include="..\Microsoft.NET.Build.Tasks\ConflictResolution\*.cs">
+      <Link>ConflictResolution\%(FileName)%(Extension)</Link>
+    </Compile>
+    <EmbeddedResource Include="..\Microsoft.NET.Build.Tasks\Resources\Strings.resx">
+      <Link>Resources\%(FileName)%(Extension)</Link>
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>Strings.Designer.cs</LastGenOutput>
+      <CustomToolNamespace>Microsoft.NET.Build.Tasks</CustomToolNamespace>
+    </EmbeddedResource>
+    <EmbeddedResource Include="..\Microsoft.NET.Build.Tasks\Resources\Strings.*.resx">
+      <Link>Resources\%(FileName)%(Extension)</Link>
+      <DependentUpon>Strings.resx</DependentUpon>
+    </EmbeddedResource>
```

Comment:
OK, that should work. Arrange for UpdateXlfOnBuild to always be false for this project after merging my change. Slap a TreatAsLocal=UpdateXlfOnBuild so that build /p:UpdateXlfOnBuild=true still works on the command line. Do you know if ResXFileGenerator -> Designer.cs can have the same problem (I vaguely recall that it can). We should probably exclude that metadata too.